dockerhub/mondoolabs/mondoo.md

```diff
@@ -1,5 +1,7 @@
 ![Mondoo](https://mondoo.com/docs/img/mondoo.logo.svg)
 
+> This is a legacy repo. We recommend using [mondoo/client](https://hub.docker.com/r/mondoo/client) instead.
```

Comment:
not sure if it was intentional that such a remark was not present in the old repo. I think we should have something because as a new user it is very confusing